components/index.js

```diff
@@ -150,23 +150,23 @@ export { default as MarkerClusterer } from './molecules/map/dynamicLocation/clus
 export { default as InfoWindow } from './molecules/map/dynamicLocation/infoWindow'
 export { default as StaticLocation } from './molecules/map/staticLocation/staticLocation'
 export { default as Navbar } from './molecules/navbar/navbar'
-export { default as PieChart } from './molecules/charts/nivo/pieChart/pieChart'
+// export { default as PieChart } from './molecules/charts/nivo/pieChart/pieChart'
```

Comment:
@Ojaghi1991 No commments

Reply:
@iwarner Do you mean remove PieChart? because there is no piechart in project